server/services/visitSessionsService.ts

```diff
@@ -389,7 +329,7 @@ export default class VisitSessionsService {
       startTimestamp: visit.startTimestamp,
       visitDate: prisonerDateTimePretty(visit.startTimestamp),
       visitTime,
-      visitRestriction: visit.visitRestriction,
+      visitRestriction: visit.visitRestriction === 'OPEN' ? 'OPEN' : 'CLOSED',
```

Comment:
This is because imported visits are sometimes UNKNOWN, not sure what we should default to tbh.

Reply:
Have asked the question, this may change to OPEN

Reply:
Similar to what's on https://dsdmoj.atlassian.net/browse/VB-995 - I raised the question of what to do with 'Unknown' visits. Feels like that issue should be linked to VB-927?

Reply:
@tpmcgowan yep, Phoebe is going to link them as it all relates to how to display them.